src/psyclone/tests/nemo/nemo_gen_test.py

```diff
@@ -55,3 +55,13 @@ def test_api_no_alg():
                         api="nemo")
     assert alg is None
     assert isinstance(psy, fparser.two.Fortran2003.Program)
+
+
+def test_non_ascii_char(parser, tmpdir):
+    ''' Check that we generate the PSy layer OK when the original Fortran
+    code contains non-ASCII characters. '''
```

Comment:
Perhaps change test name and description (and example file name) to say original code contains utf8, rather than non-ascii, as this fix works for utf, not any other encoding.

Reply:
Done.

Reply:
Great.